haxxx/validation/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM fedora:31
+
+RUN dnf install -y python-ogr python-copr python-copr-doc pip && dnf clean all
```

Do we really need `python-copr-doc`?

```suggestion
RUN dnf install -y python3-ogr python3-copr python3-pip && dnf clean all
```